### ISSUE TITLE
lxqt-config-input crashes on wayland

### ISSUE THREAD

**Issue by stefonarch** [lxqt/lxqt-config#949]
```
#0  0x00007f3355e36794 in XQueryExtension () from /usr/lib/libX11.so.6
[Current thread is 1 (Thread 0x7f335340ed40 (LWP 229246))]
(gdb) where
#0  0x00007f3355e36794 in XQueryExtension () at /usr/lib/libX11.so.6
#1  0x00007f3355d865c2 in  () at /usr/lib/libXi.so.6
#2  0x00007f3355d8ed95 in XIQueryDevice () at /usr/lib/libXi.so.6
#3  0x0000559207eca1f4 in TouchpadDevice::find_xi2_device() (this=0x7ffcc23c2790) at /home/stef/git/lxqt/lxqt-config/lxqt-config-input/touchpaddevice.cpp:229
#4  0x0000559207ec9ec4 in TouchpadDevice::enumerate_from_udev() () at /home/stef/git/lxqt/lxqt-config/lxqt-config-input/touchpaddevice.cpp:201
#5  0x0000559207ec6432 in TouchpadConfig::TouchpadConfig(LXQt::Settings*, QWidget*) (this=0x55920847efe0, _settings=0x7ffcc23c2b10, parent=0x7ffcc23c2ba0)
    at /home/stef/git/lxqt/lxqt-config/lxqt-config-input/touchpadconfig.cpp:33
#6  0x0000559207eb6c09 in main(int, char**) (argc=1, argv=0x7ffcc23c2d38) at /home/stef/git/lxqt/lxqt-config/lxqt-config-input/lxqt-config-input.cpp:111



```
It is usable (=will write settings to `session.conf`) if opened with `QT_QPA_PLATFORM=xcb lxqt-config-input`

##### Context

Changed settings like keyboard layout or pointer speed/accel_factor   in all of the wl-roots based compositors I'm using can't be applied without restarting the session. 

For labwc I'm working on a import script that reads some lines from  `session.conf` and writes settings to `labwc/environment`. This script is called from the `startlxqtlabwc` script after login. Atm for each compositor a slightly different script is needed.

##### System Information

* Distribution & Version:  git. arch, all

**Comment by stefonarch** [comment]
Looks like solved by libkscreen 6.x now.

**Comment by stefonarch** [comment]
Fixed by 5b63110bc0de3ed64b218113110593f76301be58
